server/src/infra/repositories/asset.repository.ts

```diff
@@ -49,6 +50,10 @@ export class AssetRepository implements IAssetRepository {
     await this.exifRepository.upsert(exif, { conflictPaths: ['assetId'] });
   }
 
+  async writeExif(pathMetadata: string, exifModify: Partial<Tags>): Promise<void> {
+    await exiftool.write(pathMetadata, exifModify);
+  }
```

Comment:
Does exiftool append to the sidecar file or is it a full replace/overwrite? If it's just an append, can we forego the read from the old sidecar in `asset.service.ts`?

Reply:
Indeed exiftool after verification append to the sidecar and not overwrite so there is no need to read and also no need to check for the non writable tag as the tag write are hardcoded